File: Topic Based Problems/01 Fundamentals/Beginner/5 Rectangle Perimeter/Java/RectanglePerimeter.java
/**
 * @file RectanglePerimeter.java
 * @author SharifdotG (sharifmdyousuf007@gmail.com)
 * @brief Rectangle Perimeter in Java
 * @version 1.0
 * @date 2023-09-24
 * 
 * @copyright Copyright (c) 2023
 * 
 */

import java.util.Scanner;

public class RectanglePerimeter {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int length, width;

        System.out.print("Enter length: ");
        length = scanner.nextInt();

        System.out.print("Enter width: ");
        width = scanner.nextInt();

        System.out.println("Perimeter of rectangle is: " + 2 * (length + width));

        scanner.close();
    }
}

File: Topic Based Problems/01 Fundamentals/Beginner/5 Rectangle Perimeter/C++/RectanglePerimeter.cpp
/**
 * @file RectanglePerimeter.cpp
 * @author SharifdotG (sharifmdyousuf007@gmail.com)
 * @brief Rectangle Perimeter in C++
 * @version 1.0
 * @date 2023-09-24
 * 
 * @copyright Copyright (c) 2023
 * 
 */

#include <iostream>

using namespace std;

int main() {
    int length, width;

    cout << "Enter length: ";
    cin >> length;

    cout << "Enter width: ";
    cin >> width;

    cout << "Perimeter of rectangle is: " << 2 * (length + width) << endl;

    return 0;
}